### PR TITLE
infra(make): Add `generate_data` target for running generator scripts

### PR DESCRIPTION
**Pull request by straight-shoota**
This target makes it easy to run all generator scripts at once. I want to add this as a check to the release process.

### CODE REVIEW

**Review by HertzDevil — COMMENTED** [review]
I think `scripts/generate_windows_zone_names.cr` is the only script that cannot work on Windows (i.e. cannot be called inside `Makefile.win`), because it still requires live tzdata on the host running the script

**Comment by straight-shoota** [comment]
I think it should be possible to have that working on Windows with a copy of tzdata.

**Comment by HertzDevil** [comment]
By the way, if we want to avoid tzdata entirely, the standard abbreviation names are also in CLDR, like this: https://github.com/unicode-org/cldr/blob/b24157695397c4c3d2d058c201da92d10c9d9022/common/main/en_NZ.xml#L165

File: Makefile.win
Comment (HertzDevil) [inline Makefile.win:1]: `Makefile.win` should use `\` as the directory separator consistently

**Review by HertzDevil — CHANGES_REQUESTED** [review]

**Review by HertzDevil — APPROVED** [review]

**Pull request merged**